PipelinesTasks/PRMetrics/buildAndReleaseTask/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -0,0 +1,156 @@
+  "loc.messages.metrics.inputs.disablingTestFactor.comment": "The message printed when the test factor validation is being disabled.",
+
+  "loc.messages.metrics.inputs.settingBaseSize": "Setting the base size input to '%s'.",
+  "loc.messages.metrics.inputs.settingBaseSize.comment": "The message printed when the base size input is being set to the user-provided value. The placeholder will be replaced by the default value.",
+
+  "loc.messages.metrics.inputs.settingCodeFileExtensions": "Setting the code file extensions input to '%s'.",
+  "loc.messages.metrics.inputs.settingCodeFileExtensions.comment": "The message printed when the code file extensions input is set to the user-provided value. The placeholder will be replaced by the default value.",
+
+  "loc.messages.metrics.inputs.settingFileMatchingPatterns": "Setting the file matching patterns input to '%s'.",
+  "loc.messages.metrics.inputs.settingFileMatchingPatterns.comment": "The message printed when the file matching patterns input is set to the user-provided value. The placeholder will be replaced by the default value.",
+
+  "loc.messages.metrics.inputs.settingGrowthRate": "Setting the growth rate input to '%s'.",
+  "loc.messages.metrics.inputs.settingGrowthRate.comment": "The message printed when the growth rate input is set to the user-provided value. The placeholder will be replaced by the default value.",
+
+  "loc.messages.metrics.inputs.settingTestFactor": "Setting the test factor input to '%s'.",
+  "loc.messages.metrics.inputs.settingTestFactor.comment": "The message printed when the test factor input is set to the user-provided value. The placeholder will be replaced by the default value.",
+
+  "loc.messages.pullRequests.pullRequest.addDescription": "❌ **Add a description.**",
+  "loc.messages.pullRequests.pullRequest.addDescription.comment": "The description to add to the pull request if none exists.",
+
+  "loc.messages.pullRequests.pullRequest.titleFormat": "%s ◾ %s",
+  "loc.messages.pullRequests.pullRequest.titleFormat.comment": "The title format. The first placeholder will be replaced by the title size and test coverage prefix, and the second placeholder will be replaced by the original pull request title.",
+
+  "loc.messages.pullRequests.pullRequestComments.commentFooter": "[Metrics computed by PR Metrics. Add it to your Azure DevOps builds!](https://marketplace.visualstudio.com/items?itemName=ms-omex.prmetrics)",
+  "loc.messages.pullRequests.pullRequestComments.commentFooter.comment": "The footer of the comment added to pull requests.",
+
+  "loc.messages.pullRequests.pullRequestComments.commentTitle": "# Metrics for iteration %s",
+  "loc.messages.pullRequests.pullRequestComments.commentTitle.comment": "The title of the comment added to pull requests. The placeholder will be replaced by the number of the pull request iteration.",
+
+  "loc.messages.pullRequests.pullRequestComments.fileIgnoredComment": "❗ **This file may not need to be reviewed.**",
```

Comment:
Based on this description, I'd suggest updating to something like this "This file doesn't need review and can be ignored."

Reply:
Ignored means two things in the system, but I've changed things around now so that we have a better name and less confusion. I've also updated the comment to make it more decisive – it was weak before because it was doing two things, but that has changed.